conf/machine/raspberrypi5.conf

```diff
@@ -0,0 +1,30 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi 5 Development Board (64bit)
+#@DESCRIPTION: Machine configuration for the RaspberryPi 5 in 64 bits mode
+
+require conf/machine/include/arm/armv8-2a/tune-cortexa76.inc
+include conf/machine/include/rpi-base.inc
+
+MACHINE_FEATURES += "pci"
+MACHINE_EXTRA_RRECOMMENDS += "\
+    linux-firmware-rpidistro-bcm43455 \
+    bluez-firmware-rpidistro-bcm4345c0-hcd \
+    linux-firmware-rpidistro-bcm43456 \
+    bluez-firmware-rpidistro-bcm4345c5-hcd \
+"
+
+RPI_KERNEL_DEVICETREE = " \
+    broadcom/bcm2712-rpi-5-b.dtb \
+"
+
+SDIMG_KERNELIMAGE ?= "kernel_2712.img"
+SERIAL_CONSOLES ?= "115200;ttyAMA10"
+
+VC4DTBO ?= "vc4-kms-v3d"
```

Comment:
Since you have added that following line,

> overlays/vc4-kms-v3d-pi5.dtbo \

I guess you might update that one with `VC4DTBO ?= "vc4-kms-v3d-pi5"`

Reply:
Okay, it makes sense. thanks for you reply.

Note: Since this `VC4DTBO ?= "vc4-kms-v3d"` is the default value and unless it is used by other recipes, adding it looks like garbage to me. Maybe it could be addressed in another PR, with all the machines.